src/odemis/driver/xt_client.py

```diff
@@ -1338,6 +1339,40 @@ def brightness_info(self):
             self.server._pyroClaimOwnership()
             return self.server.brightness_info()
 
+    def start_autostig(self, n_images=17, sweep_range=0.015, dwell_time=3e-6, resolution=(1536, 1024),
```

Comment:
Could you move it next to `start_autostig_flash`? That would be clearer that they are alternatives.